<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <widget class="QWidget" name="camWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="Figure" name="distanceWidget" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>300</width>
     <height>180</height>
    </rect>
   </property>
  </widget>
  <widget class="Figure" name="angleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>300</width>
     <height>180</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PAUSE</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>681</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Figure</class>
   <extends>QWidget</extends>
   <header location="global">figure.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>setAngle(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reset()</slot>
  <slot>pause()</slot>
  <slot>setAngle(int)</slot>
 </slots>
</ui>
